.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and release SDK 
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-18.04
+    steps:
+      # Get the code into the container
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Create a new release
+      - name: Semantic version bump
+        uses: rimonhanna/gh-action-bump-version@master
```

Comment:
this looks like a fork for an action of automatic version bump of npm packages, we should not use `package.json` for golang.